pkg/asset/installconfig/aws/permissions.go

```diff
@@ -336,6 +340,12 @@ var permissions = map[PermissionGroup][]string{
 		// Needed so the installer can use the provided custom IAM role
 		"sts:AssumeRole",
 	},
+	PermissionCreateCarrierGateway: {
```

Comment:
nit: would group name without `Create` sounds better as the permission list includes delete?:
```suggestion
	PermissionCarrierGateway: {
```